### DISCUSSION TITLE
Gesture based navigation

### DISCUSSION THREAD

**Discussion by Kamil-H** [bumble-tech/appyx#301] [Ideas]
Are you considering adding gesture-based navigation? I'm very interested in implementing Slack-like navigation where the user can dismiss the screen with a simple swipe from left to right. That would be very similar to what [Fragula](https://github.com/massivemadness/Fragula) enables. I've seen that #103 has been closed, but do you consider investigating this option?

**Comment by KovalevAndrey** [comment]
Hi @Kamil-H, yes https://github.com/bumble-tech/appyx/pull/103 was closed as we decided to do it the other way. We're currently working on the idea of navigating with gestures, please stay tuned!

**Comment by Kamil-H** [comment]
Great to hear that! Looking for what you will come up with 🤞

**Comment by zsoltk** [comment]
Hey @Kamil-H gesture control is now available in our 2.0-alpha. You can check it in: https://bumble-tech.github.io/appyx/

**Reply by Kamil-H** [reply]
Neat! Thank you for the information! 🙂
